docs/config.md

```diff
@@ -342,10 +342,18 @@ Tells reprovider what should be announced. Valid strategies are:
 
 Options for configuring the swarm.
 
-- `AddrFilters`
-An array of address filters (multiaddr netmasks) to filter dials to.
-See [this issue](https://github.com/ipfs/go-ipfs/issues/1226#issuecomment-120494604) for more
-information.
+- `AddrFilters` An array of address filters (multiaddr netmasks) to filter
+automatic dials to. The ipfs daemon will otherwise attempt to randomly dial
+the default listen port on every IP of the the local network IP range. This
+improves autodiscovery of other daemons in the same network, but may also
+trigger netscan alerts on some hosting providers or cause strain in some
+setups.
```

Comment:
This should explain _why_ we do that and how. That is, we're not intentionally dialing every address one-by-one.

(Otherwise, we'll get even more questions)

Reply:
Isn't the *why* to improve autodiscovery? I don't have much idea about the *how*.. is it random?

Reply:
No. We end up dialing local addresses because peers publish these addresses in the DHT. We aren't intentionally scanning the local network.

Reply:
Ah, will update accordingly. Thanks!